modules/tensor_mechanics/examples/bridge/bridge_large_strain.i

```diff
@@ -68,11 +64,14 @@
 []
 
 [BCs]
-  [./PressureTM]
+  [./Pressure]
     [./load]
       #Applies the pressure
       boundary = top
       factor = 5e5 # Pa
+      disp_x = disp_x
```

Comment:
This is just a side note, but we need to update ```PressureAction``` to use the ```displacements``` parameter style as well.